sdk/core/azure-core-http-vertx/src/main/java/com/azure/core/http/vertx/VertxAsyncHttpClient.java

```diff
@@ -117,9 +117,4 @@ public Mono<HttpResponse> send(HttpRequest request, Context context) {
             }
         }));
     }
-
-    @Override
```

Comment:
This change is not doing anything new. The base class calls .block() by default.